<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>55</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Pomodoro</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>300</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="topBar" native="true">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>280</width>
           <height>20</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
	background-color:  #282a36;
	color:  #f8f8f2;
	border-radius: 10px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="btnIcon">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	background-color: rgba(0,0,0,0);
}</string>
               </property>
               <property name="icon">
                <iconset resource="mainwindow.qrc">
                 <normaloff>:/Logo/assets/Pomodoro.ico</normaloff>:/Logo/assets/Pomodoro.ico</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnOpenSettings">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	width: 100px;
	height: 25px;
	color: rgb(151, 159, 200);
	background-color: rgb(68, 71, 90);
	font: 18px;
	border: none;
}
      
QPushButton:hover:!pressed {
	background-color: rgb(151, 159, 200);
	color:  rgb(68, 71, 90);
}

QPlainTextEdit {
	border: #fff;
}</string>
               </property>
               <property name="text">
                <string>S</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="lbTitle">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <italic>false</italic>
              <bold>true</bold>
              <stylestrategy>PreferDefault</stylestrategy>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 9pt;</string>
            </property>
            <property name="text">
             <string>Pomodoro - zeroproject</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="cbAlwaysOnTop">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>15</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>15</width>
              <height>20</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox::indicator:checked {
	background-color: #ff79c6;
	border: none;
}
QCheckBox::indicator:unchecked{
	background-color: rgba(189, 147, 249, 145);
	border: none;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnMinimize">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #282a36;
	color: #f8f8f2;
	border-radius: 10px;
	font-size: 14px;
}</string>
            </property>
            <property name="text">
             <string>_</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnExit">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #282a36;
	color: #f8f8f2;
	border-radius: 10px;
	font-size: 14px;
}</string>
            </property>
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="container">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>280</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>280</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="progressBar" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QFrame" name="circle_bg">
            <property name="styleSheet">
             <string notr="true">QFrame {
	background-color:  #282a36;
	color:  #f8f8f2;
	border-radius: 120px;
	font: 16pt &quot;Segoe UI&quot;;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="texts">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>160</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background:none;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <layout class="QGridLayout" name="gridLayout">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <item row="2" column="0" alignment="Qt::AlignHCenter">
                   <widget class="QFrame" name="buttons">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QPushButton" name="actionButton">
                       <property name="maximumSize">
                        <size>
                         <width>30</width>
                         <height>30</height>
                        </size>
                       </property>
                       <property name="cursor">
                        <cursorShape>PointingHandCursor</cursorShape>
                       </property>
                       <property name="layoutDirection">
                        <enum>Qt::LeftToRight</enum>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QPushButton {
	width: 100px;
    height: 25px;
    color: rgb(151, 159, 200);
    background-color: rgb(68, 71, 90);
    font-size: 18px;
    border-radius: 12px;
	margin-right: 5px;
}
    
QPushButton:hover:!pressed {
    background-color: rgb(151, 159, 200);
    color:  rgb(68, 71, 90);
}</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                       <property name="icon">
                        <iconset resource="mainwindow.qrc">
                         <normaloff>:/Icons/assets/play.png</normaloff>:/Icons/assets/play.png</iconset>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="stopButton">
                       <property name="maximumSize">
                        <size>
                         <width>30</width>
                         <height>30</height>
                        </size>
                       </property>
                       <property name="cursor">
                        <cursorShape>PointingHandCursor</cursorShape>
                       </property>
                       <property name="layoutDirection">
                        <enum>Qt::LeftToRight</enum>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QPushButton {
	width: 100px;
    height: 25px;
    color: rgb(151, 159, 200);
    background-color: rgb(68, 71, 90);
    font-size: 18px;
    border-radius: 12px;
	margin-left: 5px;
}
    
QPushButton:hover:!pressed {
    background-color: rgb(151, 159, 200);
    color:  rgb(68, 71, 90);
}</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                       <property name="icon">
                        <iconset resource="mainwindow.qrc">
                         <normaloff>:/Icons/assets/stop.png</normaloff>:/Icons/assets/stop.png</iconset>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item row="0" column="0" alignment="Qt::AlignHCenter">
                   <widget class="QLabel" name="lbTimeTitle">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>Segoe UI</family>
                      <pointsize>16</pointsize>
                      <italic>false</italic>
                      <bold>false</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>Work Time</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0" alignment="Qt::AlignHCenter">
                   <widget class="QFrame" name="time">
                    <property name="minimumSize">
                     <size>
                      <width>130</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>130</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLCDNumber" name="minutes">
                       <property name="minimumSize">
                        <size>
                         <width>60</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>50</height>
                        </size>
                       </property>
                       <property name="digitCount">
                        <number>2</number>
                       </property>
                       <property name="intValue" stdset="0">
                        <number>100</number>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="separator">
                       <property name="maximumSize">
                        <size>
                         <width>10</width>
                         <height>42</height>
                        </size>
                       </property>
                       <property name="font">
                        <font>
                         <family>Segoe UI</family>
                         <pointsize>31</pointsize>
                         <italic>false</italic>
                         <bold>false</bold>
                        </font>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">font: 31pt;</string>
                       </property>
                       <property name="text">
                        <string>:</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLCDNumber" name="seconds">
                       <property name="minimumSize">
                        <size>
                         <width>60</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>50</height>
                        </size>
                       </property>
                       <property name="smallDecimalPoint">
                        <bool>false</bool>
                       </property>
                       <property name="digitCount">
                        <number>2</number>
                       </property>
                       <property name="intValue" stdset="0">
                        <number>100</number>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
